README.md

```diff
@@ -37,11 +37,6 @@ $output = "$NXLOGDIR\cert\ca.pem"
 
 (New-Object System.Net.WebClient).DownloadFile($url, $output)
 ```
-
-### Modify LogDNA NXLog Configuration
```

Comment:
Why did you remove this part?

Reply:
This is duplicate to this: https://github.com/logdna/logdna-nxlog/blob/master/README.md#copy-logdna-nxlog-configuration